components/pricing/PricingCard.vue

```diff
@@ -7,12 +7,11 @@
           {{ plan.description }}
         </p>
         <div class="d-flex align-items-baseline">
-          <span
-            v-if="plan.type === 'Yearly'"
-            class="pricing__prev__amount me-2"
-            >{{ plan.prevPrice }}</span
-          >
-          <span class="pricing__amount">${{ plan.price }}</span>
+          <span v-if="false" class="pricing__striked me-2">
+            {{ plan.prevPrice }}
```

Comment:
Previous price??

Reply:
That is `v-if="false"`, kept it there so that we can quickly run sales going ahead.